backend/app.js

```diff
@@ -0,0 +1,68 @@
+const express = require("express");
+const cors = require("cors");
+const bodyParser = require("body-parser")
+const time = require("express-timestamp")
+
+const School = require("./School")
+
+port = 5000;
+const app = express();
+app.use(bodyParser.urlencoded({extended: false}))
+app.use(express.json());
+app.use(cors())
+app.use(time.init)
+
+let mySchool = new School();
+console.log(mySchool)
+console.log(mySchool)
+mySchool.enrollStudent("algebra")
+console.log(mySchool)
+mySchool.getStudentsByClass("algebra")
+
+app.get('/', (req, res) => {
+    console.log('youve hit GET/ endpoint', mySchool)
+    res.json({
+        message: 'youve hit GET/ endpoint',
+        school: mySchool,
+    })
+})
+
+app.post("/class", (req,res) => {
+    mySchool.addClass(req.body.classname, req.body.teacher)
+    console.log(req.body)
+    res.json({ 
+        "class": { 
+            "name": req.body.classname, 
```

Comment:
You are returning your request inside of your response here.It's not technically correct. Your response should come 100% from your backend. You can assign the key of `class` to the value of your new variable `newClass` we've just created. That way when you parse your response on the front end you can key into res.data.class and get back the full object of your newly created class with the name and the teacher.